apis/assessmentapi/assessmentapi.yaml

```diff
@@ -33,8 +33,9 @@ paths:
     post:
       description: >-
           This API is used for creating an assessment Item.
-            - The **_/create_** endpoint executes the "Create Assessment Item" request based on parameters provided as metadata in the request body.
-            - Required fields cannot be left null or empty
+            - The Sunbird API endpoint for Create Assessment Item is `/org/v1/create`

```

Comment:
Actually, we can be less verbose. We can just say **Request Path: "/item/v1/create"**. Also the endpoint details are incorrect, it says '/org/v1/create' instead of '/item/v1/create'. Hope it will be corrected.